lib/system/comparisons.nim

```diff
@@ -252,11 +252,12 @@ proc max*[T](x: openArray[T]): T =
 
 
 proc clamp*[T](x, a, b: T): T =
-  ## Limits the value ``x`` within the interval [a, b].
-  ##
-  ## .. code-block:: Nim
-  ##   assert((1.4).clamp(0.0, 1.0) == 1.0)
-  ##   assert((0.5).clamp(0.0, 1.0) == 0.5)
+  ## Limits the value `x` within the interval [a, b].
+  ## This functions is equivalent to `max(a, min(b, x))`.
```

Comment:
```suggestion
  ## This function is equivalent to `max(a, min(b, x))`.
```
...or replace "function" with "proc" (doesn't matter to me either way)

Reply:
or even (shorter is better, the fact that it's a proc/func/template doesn't matter here):
This is equivalent ...